README.md

```diff
@@ -1,107 +1,115 @@
+<p align="center">
+<img src="https://user-images.githubusercontent.com/47296334/199617207-1140698a-4d53-462d-9bc7-303d14be060b.png" width="800">
+</p>
+There are several native libraries we may offload. Currently we are working on Clickhouse and Velox as native backend. Velox is a C++ database acceleration library which provides reusable, extensible, and high-performance data processing components. More details can be found from https://github.com/facebookincubator/velox/. We also implemented a basic backend using Arrow Computer Engine which is for reference only. Gluten can also be easily extended to any accelerator libraries as backend.
 
-Not all the operators and functions are added. 22 queries of TPCH can pass. We are working on TPCDS queries. <b> We haven't pass Spark UTs, the work is just started and expected to finish in Q4 2022. So any other queries except H/DS may run into issues. Feel free to submit it to our issue list.</b>
+There are several key component in GLuten:
+* Query plan conversion which convert Spark's physical plan into substrait plan in each stage.
+* Unified memory management in Spark is used to control the native memory allocation as well
+* Columnar shuffle is used to shuffle columnar data directly. The shuffle service still reuse the one in Spark core. The exchange operator is reimplemented to support columnar data format
+* For unsupported operators or functions Gluten fallback the operator to Vanilla Spark. There are C2R and R2C converter to convert the columnar data and Spark's internal row data. Both C2R and R2C are implemented natively as well
+* Metrics are very important to get insight of Spark's execution, identify the issues or bottlenecks. Gluten collects the metrics from native library and shows in Spark UI.
+* Shim layer is used to support mutiple releases of Spark. Gluten does not and will not support all the Spark releases. The plan is to support the latest 2-3 spark stable release only.
 
-Not all data type is supported now. Refere to Velox and Clickhouse to check the supported data types.
+# Usage
 
-Operator stat info is pretty useful to understand Spark's execution status. Velox metrics are already added to Spark metric list for Velox backend. More info about Velox's stat can be found here: https://facebookincubator.github.io/velox/develop/debugging/print-plan-with-stats.html
+Gluten is still under actively developping now. There isn't released binary yet. The only way to use Gluten is to build from source.
```

Comment:
nit. fixing some sentences: 
Gluten is still under active development now. There isn't a released binary yet. The only way to use Gluten is to build from source.